src/Microsoft.AspNetCore.Mvc.Core/Internal/IControllerArgumentBinder.cs

```diff
@@ -1,28 +0,0 @@
-// Copyright (c) .NET Foundation. All rights reserved.
-// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
-
-using System.Collections.Generic;
-using System.Threading.Tasks;
-
-namespace Microsoft.AspNetCore.Mvc.Internal
-{
-    /// <summary>
-    /// Provides a dictionary of action arguments.
-    /// </summary>
-    public interface IControllerArgumentBinder
```

Comment:
It's internal API so it wasn't particularly useful to start with. Plus it's wrapping a for loop now which is not very useful.